TowerFox/Base.lproj/Main.storyboard

```diff
@@ -454,7 +454,7 @@
                                                                     </userDefinedRuntimeAttributes>
                                                                 </textField>
                                                             </subviews>
-                                                            <color key="backgroundColor" name="App-onprimary-color"/>
+                                                            <color key="backgroundColor" name="App-textfiled-background"/>
```

Comment:
I guess, this is for the bottom button background in review screen,
do not use textfiled-background, add a new asset, and use color for dark mode as #0F0F0F
Also, fix the typo `textfiled-background`